english/hermione/hermioneHere

```diff
@@ -0,0 +1,2 @@
+Hermione is always getting into trouble with Ron and Harry.  They
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.